### DISCUSSION TITLE
Unable to check the SLCs file

### DISCUSSION THREAD

**Discussion by DinhTaiNguyen** [alexisInSAR/EZ-InSAR#53] [Help]
Hi Alexis,

I hope this discussion finds you well. I am a Windows user and I recently completed the setup of EZ-InSAR using Ubuntu 20.04, following the steps you provided. I successfully checked the echo and loaded the environment using "load_insar".

I then proceeded to open the GUI as instructed in the manual. I made sure to correctly specify the "path" and the "pass" in the "Selection of study area" section. However, I encountered an issue when trying to check the SLCs.

**Could you please guide me on what steps to take next to resolve this issue?** (It seems that I am unable to download the SLCs from the website???. I have already updated the pathinformation.txt file. )

Thank you for your assistance.

Kind regards,
Tai

This is the bugs
![image](https://github.com/alexisInSAR/EZ-InSAR/assets/103304398/e433c485-b36e-4d18-a291-c41e1cbb0378)
![image](https://github.com/alexisInSAR/EZ-InSAR/assets/103304398/52c7e6fd-bcab-4b78-9a0b-ab91aed14b39)
![image](https://github.com/alexisInSAR/EZ-InSAR/assets/103304398/cd2c8ca1-832a-4370-853a-b0ae481ba5da)
![image](https://github.com/alexisInSAR/EZ-InSAR/assets/103304398/915e9b64-0d62-4054-aefd-ba7620c3fee1)
![image](https://github.com/alexisInSAR/EZ-InSAR/assets/103304398/c8d9eafe-4f71-45cc-a6d1-68460f7cf538)

**Comment by alexisInSAR** [comment]
Can you confirm that you have installed EZ-InSAR using WLS?

Kind regards, 

Alexis

**Reply by DinhTaiNguyen** [reply]
Yes. This one.  Ubuntu 20.04
![image](https://github.com/alexisInSAR/EZ-InSAR/assets/103304398/61fed13b-1ad9-46c1-89a1-04bce809f1b1)

**Comment by alexisInSAR** [comment]
I assume that you have used the documentation from [here](https://github.com/alexisInSAR/EZ-InSAR_For_Windows). It seems that you have two errors (perphas the second issue is linked to the first error):
1. The linux commands are not recognised in MATLAB; 
2. The list is therefore not created and the createlistSLC.m function fails. 

I have not created support for Windows, but I am sure @insarwxw could help you.  

Kind regards, 

Alexis

**Comment by riccardol9** [comment]
hi @DinhTaiNguyen i'm having the exact same problem. Did you manage to solve it? If yes, how?

King regards,

Riccardo

**Reply by alexisInSAR** [reply]
Hi all, 

I will take a look.

Kind regards, 

Alexis
